PSReadLine/Completion.cs

```diff
@@ -24,6 +26,10 @@ public partial class PSConsoleReadLine
         private CommandCompletion _tabCompletions;
         private Runspace _runspace;
 
+        private static System.Management.Automation.PowerShell _ps;
+
+        private static Microsoft.PowerShell.Pager _pager;
```

Comment:
I think they don't need to be static. Just allocate them in your key handler.

Reply:
Fixed.